skglm/utils/sparse_ops.py

```diff
@@ -92,3 +92,11 @@ def _XT_dot_vec(X_data, X_indptr, X_indices, vec):
             result[j] += X_data[idx] * vec[X_indices[idx]]
 
     return result
+
```

Comment:
@jasperlamm the code guidelines that we follow expect two blank lines between function definitions.

You can run `flake8` locally to check flake8 violations without having to go through the continuous integration.